src/MusicalScore/Graphical/VexFlow/CanvasVexFlowBackend.ts

```diff
@@ -52,11 +52,22 @@ export class CanvasVexFlowBackend extends VexFlowBackend {
         this.canvasRenderingCtx.fillText(text, screenPosition.x, screenPosition.y + heightInPixel);
         this.canvasRenderingCtx.font = old;
     }
-    public renderRectangle(rectangle: RectangleF2D, styleId: number): void {
+
+    /**
+     * Renders a rectangle with the given style to the screen.
+     * It is given in screen coordinates.
+     * @param rectangle the rect in screen coordinates
+     * @param layer is the current rendering layer. There are many layers on top of each other to which can be rendered. Not needed for now.
+     * @param styleId the style id
+     * @param alpha alpha value between 0 and 1
+     */
+    public renderRectangle(rectangle: RectangleF2D, styleId: number, alpha: number = 1): void {
```

Comment:
Hopefully the last annoying comment now, I think you can omit the docstring here, as it will be inherited from the parent class(es). Typedoc should do so, IntelliSense in Visual Studio Code as well. Could you check that? If not, you might be able to put a `@inheritdoc`, but it should never be required to have the same docstring twice. So I guess, just put the comment in `VexFlowBackend` and everything should be fine?